cpu/sensors.py

```diff
@@ -33,6 +33,8 @@ class Sensors(Test):
 
     """
     Run sensors command on ppc architectures.
+
+    :avocado: tags=cpu,power,privileged
```

Comment:
@clebergnu sorry may be doc string having issue will fix ,
*  checking `CONFIG_SENSORS_IBMPOWERNV` config only i think when distro match with ppc please have a look in this 

https://github.com/avocado-framework-tests/avocado-misc-tests/blob/master/cpu/sensors.py#L69